plenum/client/client.py

```diff
@@ -80,8 +85,9 @@ def __init__(self,
         :param ha: tuple of host and port
         """
         self.config = config or getConfig()
-        self.basedirpath = basedirpath or os.path.join(self.config.baseDir,
-                                                       self.config.NETWORK_NAME)
+        self.basedirpath = basedirpath or self.config.baseDir
```

Comment:
Do we need to have basedir here?

Reply:
Yes, we decided to leave basedir for client earlier, there are several reasons for it. General reason is replacing of python-based client with IndySDK-based client, so that decision to leave basedir reduces re-factoring time.